README.md

```diff
@@ -0,0 +1,32 @@
+# Telemetry-Logstash
+
+A tool that aggregates and structures logs of the cluster's components
+
+# Logstash Helm Chart
```

Comment:
Would you like to follow this structure for this section? 

## Usage

[Helm](https://helm.sh) must be installed to use the charts.
Please refer to Helm's [documentation](https://helm.sh/docs/) to get started.

- mongoDB · bitnami/mongodb (https://artifacthub.io/packages/helm/bitnami/mongodb)

Once Helm is set up properly, add the repo as follows: